### DISCUSSION TITLE
Folder names on the file system

### DISCUSSION THREAD

**Discussion by metaeaux** [pixelspark/sushitrain#8] [Q&A]
Thanks for putting together such a polished syncthing client for iOS! I’ve been testing it out and have had a great experience so far. 

One difference with the syncthing clients is the folder names on the file system are different. For example, if I sync the default folder from other clients to synctrain, the folder name ends up being “default” instead of “Sync”. Would it be possible to customise folder names? Juggling different names on different clients adds a bit of cognitive burden to navigating files.

**Comment by pixelspark** [comment]
Thanks for the kind words!

By default the folder names on the file system are equal to the folder ID (the 'Sync' folder has folder ID 'default'). 

If you want the folder on disk to have a different name, use the feature to add an external folder (you should be able to pick your own name and place it under "On my iPhone").
